package.json

```diff
@@ -57,29 +57,29 @@
   "author": "ethdev.com",
   "authors": [
     {
-      "name": "Jeffery Wilcke",
-      "email": "jeff@ethdev.com",
-      "url": "https://github.com/obscuren"
-    },
-    {
       "name": "Marek Kotewicz",
       "email": "marek@ethdev.com",
       "url": "https://github.com/debris"
     },
     {
+      "name": "Fabian Vogelsteller",
+      "email": "fabian@ethdev.com",
+      "homepage": "http://frozeman.de"
+    },
```

Comment:
I just ordered it in the order, of contributors, no reason that jeff is on the top here :)